pkg/pkg.ml

```diff
@@ -0,0 +1,2 @@
+#use "topfind"
+#require "topkg-jbuilder.auto"
```

Comment:
Why is this file still there?

Reply:
I'm not familiar with `topkg` and to err on side of caution chose to use the analogue from https://github.com/ocaml-ppx/ppx_deriving_yojson/blob/master/pkg/pkg.ml which looked like the jbuilder version of it.

Reply:
This is only for packages which use topkg-jbuilder. topkg-jbuilder has been superseded by dune-release which requires no boilerplate files.